cmd/brew-report-issue.rb

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env ruby
+# Creates and closes failure debugging issues on a project.
+close = !!ARGV.delete("--close")
+user_repo = ARGV.shift
+message = ARGV.shift
+
+if user_repo.to_s.empty? || message.to_s.empty?
+  abort "Usage: brew report-issue [--close] <user/repo> <message> [<STDIN piped body>]"
+end
+
+unless close
+  abort "Error: the issue/comment body should be piped over STDIN!" if STDIN.tty?
```

Comment:
Nice! I had thought about this, but didn't form the thought well enough to comment :grin: